src/Psr/CacheItemPool/CacheItemPoolDecorator.php

```diff
@@ -194,7 +194,7 @@ public function deleteItems(array $keys)
         $this->deferred = array_diff_key($this->deferred, array_flip($keys));
 
         try {
-            return null !== $this->storage->removeItems($keys);
+            return $this->storage->removeItems($keys) === [];
```

Comment:
As per https://github.com/laminas/laminas-cache-storage-adapter-apcu/pull/7#issuecomment-831262536 I think this is a BC Break, take a look at:

https://github.com/php-cache/integration-tests/blob/824633c9547dc3c7ac5faeca03dd82939cdeb73c/src/CachePoolTest.php#L303

Missing keys shouldn't result in a `return false` here.

Reply:
Then, that is a problem of the underlying storage.
`StorageInterface::removeItems` is clear on this:

https://github.com/laminas/laminas-cache/blob/15e0b44324a1dae500648ca78bd56e8671565661/src/Storage/StorageInterface.php#L196

I've recently found the same problem in `laminas-cache-storage-adapter-redis`.
The thing is, that it only worked due to the fact that the underlying cache storages had invalid return-types.

I am uncertain on how we should handle this, as there are adapters which actually do return an empty array.
`null` is never returned, it could also be `!== true` in that case.

I'll create a new issue based on your comment to see if we should revert this change and instead "fix" it only in v3.0.0 while also fixing all the adapters.